### PR TITLE
Added highlighters to docs

### PR DESCRIPTION
**Pull request by chill117**

### CODE REVIEW

File: docs/src/joint/api/dia/Paper/prototype/options/highlighting.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<p><code>highlighting</code> - Configure which highlighter to use (and with which options) for each type of interaction. For example:</p>
```
Comment (kumilingus) [inline docs/src/joint/api/dia/Paper/prototype/options/highlighting.html]: Can we add a list of all types of interactions with a short description?

**Comment by kumilingus** [comment]
Can we also document the second `option` parameter in `joint.dia.ElementView.prototype.highlight` / `joint.dia.ElementView.prototype.unhighlight` please?

File: docs/src/joint/api/dia/Paper/prototype/options/highlighting.html (hunk 1 above)
Comment (chill117) [inline docs/src/joint/api/dia/Paper/prototype/options/highlighting.html]: Sure :)

**Comment by chill117** [comment]
> Can we also document the second option parameter in joint.dia.ElementView.prototype.highlight / joint.dia.ElementView.prototype.unhighlight please?

Sure, I'll add this.

I am also going to add Cell and CellView to the docs, because `highlight()` and `unhighlight()` are actually methods on CellView, not ElementView.

**Pull request merged**